controllers/awscluster_controller.go

```diff
@@ -80,7 +74,7 @@ func (r *AWSClusterReconciler) Reconcile(req ctrl.Request) (_ ctrl.Result, reter
 		return reconcile.Result{}, nil
 	}
 
-	log = log.WithName(fmt.Sprintf("cluster=%s", cluster.Name))
+	log = log.WithValues("cluster", cluster.Name)
```

Comment:
Do we even need this given that we're working on the awsCluster field?

Reply:
This gives context back to the related Cluster. I'm not sure we should remove it.

Reply:
+1 for keeping